test/globalTeardown.ts

```diff
@@ -0,0 +1,14 @@
+export default async () => {
+  // @ts-ignore
+  const proc = global.edgedbProc;
+  // @ts-ignore
+  const conn = global.edgedbConn;
+  // tslint:disable-next-line
+  console.log("Shutting down EdgeDB test cluster...");
+
+  await conn.close();
+  proc.kill();
```

Comment:
Why an explicit `kill` if you rely on `auto-shutdown`?  Maybe `join`?

Reply:
>  if you rely on auto-shutdown

Only when you press ^C -- in which case nodejs env just craps out and dies immediately. In this case EdgeDB will shutdown itself.

>  Maybe join?

No ".join()" in nodejs APIs

I'll try removing `.kill` though...

Reply:
There's 'close' event I can probably use https://nodejs.org/api/child_process.html#child_process_child_process